File: src/kyu_6/help_the_bookseller/js/stockListTest.js
const stockList = require("./stockList");
const assert = require('chai').assert
describe("Tests", () => {
    it("test", () => {
        let b,c,res
        b = ["BBAR 150", "CDXE 515", "BKWR 250", "BTSQ 890", "DRTY 600"]
        c = ["A", "B", "C", "D"]
        res = "(A : 0) - (B : 1290) - (C : 515) - (D : 600)"
        assert.strictEqual(stockList(b, c), res)

        b = ["ABAR 200", "CDXE 500", "BKWR 250", "BTSQ 890", "DRTY 600"]
        c = ["A", "B"]
        res = "(A : 200) - (B : 1140)"
        assert.strictEqual(stockList(b, c), res)
    });
});

File: src/kyu_6/help_the_bookseller/java/StockListTest.java
package kyu_6.help_the_bookseller.java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class StockListTest {

  @Test
  public void test1() {
    String art[] = new String[]{"ABAR 200", "CDXE 500", "BKWR 250", "BTSQ 890", "DRTY 600"};
    String cd[] = new String[] {"A", "B"};
    assertEquals("(A : 200) - (B : 1140)",
            StockList.stockSummary(art, cd));
  }
}
